diff --git screensaver/CMakeLists.txt screensaver/CMakeLists.txt
index 8a904aa..be29c20 100644
--- screensaver/CMakeLists.txt
+++ screensaver/CMakeLists.txt
@@ -41,7 +41,7 @@ target_link_libraries(ukui-screensaver-default Qt5::Core Qt5::Widgets Qt5::X11Ex
 
 install(TARGETS
 	ukui-screensaver-default
-	DESTINATION lib/ukui-screensaver)
+	DESTINATION lib64/ukui-screensaver)
 
 install(FILES
         screensaver.ini 
diff --git src/configuration.cpp src/configuration.cpp
index 4ab1c64..c78f604 100644
--- src/configuration.cpp
+++ src/configuration.cpp
@@ -34,7 +34,7 @@
 #define KEY_IMAGE_TRANSITION_EFFECT "image-transition-effect"
 #define KEY_IMAGE_SWITCH_INTERVAL "image-switch-interval"
 #define KEY_BACKGROUND "background"
-#define XSCREENSAVER_DIRNAME "/usr/lib/xscreensaver"
+#define XSCREENSAVER_DIRNAME "/usr/libexec/xscreensaver"
 
 Configuration::Configuration(QObject *parent) : QObject(parent)
 {
@@ -127,7 +127,7 @@ ScreenSaver *Configuration::getScreensaver()
     case SAVER_RANDOM:
     {
         if(themes.count()==0){
-            saver->path = "/usr/lib/ukui-screensaver/ukui-screensaver-default";
+            saver->path = "/usr/libexec/ukui-screensaver/ukui-screensaver-default";
             break;
         }
         qsrand((unsigned)time(0));
@@ -156,7 +156,7 @@ ScreenSaver *Configuration::getScreensaver()
         break;
     }
     case SAVER_DEFAULE:
-        saver->path = "/usr/lib/ukui-screensaver/ukui-screensaver-default";
+        saver->path = "/usr/libexec/ukui-screensaver/ukui-screensaver-default";
         break;
     default:
         break;
@@ -174,7 +174,7 @@ QString Configuration::getXScreensaverPath(const QString &theme)
     if(QFile(filePath).exists())
         return filePath;
     else
-        return "/usr/lib/ukui-screensaver/ukui-screensaver-default";
+        return "/usr/libexec/ukui-screensaver/ukui-screensaver-default";
 }
 
 bool Configuration::ispicture(QString filepath)
diff --git src/fullbackgroundwidget.cpp src/fullbackgroundwidget.cpp
index 164afa3..1ec0ae1 100644
--- src/fullbackgroundwidget.cpp
+++ src/fullbackgroundwidget.cpp
@@ -253,7 +253,7 @@ void FullBackgroundWidget::mousePressEvent(QMouseEvent *e)
     if(screenStatus & SCREEN_SAVER)
     {
         ScreenSaver *saver = configuration->getScreensaver();
-        if(saver->path == "/usr/lib/ukui-screensaver/ukui-screensaver-default")
+        if(saver->path == "/usr/libexec/ukui-screensaver/ukui-screensaver-default")
                return ;
         clearScreensavers();
     }
@@ -467,7 +467,7 @@ void FullBackgroundWidget::onGlobalButtonDrag(int xPos, int yPos)
     if(screenStatus & SCREEN_SAVER)
     {
         ScreenSaver *saver = configuration->getScreensaver();
-        if(saver->path == "/usr/lib/ukui-screensaver/ukui-screensaver-default")
+        if(saver->path == "/usr/libexec/ukui-screensaver/ukui-screensaver-default")
                return ;
         clearScreensavers();
     }
