diff --git CMakeLists.txt CMakeLists.txt
index 176b7e3..d9910e9 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -20,7 +20,7 @@ ExternalProject_Add(peony-extensions-qmake
   SOURCE_DIR        ${peony-extensions-qmake-impl-dir}
   PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/peony-extensions-qmake
   BUILD_IN_SOURCE   1
-  CONFIGURE_COMMAND qmake ${peony-extensions-qmake-impl-dir}
+  CONFIGURE_COMMAND qmake-qt5 ${peony-extensions-qmake-impl-dir}
   BUILD_COMMAND     make
   INSTALL_COMMAND   cmake -E echo "skip install step."
   )
diff --git peony-extensions-cmake/peony-qt-admin/CMakeLists.txt peony-extensions-cmake/peony-qt-admin/CMakeLists.txt
index 6f4d3f3..1cc7122 100644
--- peony-extensions-cmake/peony-qt-admin/CMakeLists.txt
+++ peony-extensions-cmake/peony-qt-admin/CMakeLists.txt
@@ -31,5 +31,5 @@ add_library(${PROJECT_NAME} SHARED "admin-menu-plugin.cpp" ${QRC_FILES})
 
 target_link_libraries(${PROJECT_NAME} Qt5::Core Qt5::Gui Qt5::Widgets Qt5::Concurrent ${PEONY_LIBRARIES})
 
-install(TARGETS peony-qt-admin DESTINATION /usr/lib/peony-qt-extensions)
+install(TARGETS peony-qt-admin DESTINATION /usr/lib64/peony-qt-extensions)
 install(FILES org.freedesktop.peony-qt-admin.policy DESTINATION /usr/share/polkit-1/actions)
diff --git peony-extensions-cmake/peony-qt-share/CMakeLists.txt peony-extensions-cmake/peony-qt-share/CMakeLists.txt
index d1c0bce..44a93b8 100644
--- peony-extensions-cmake/peony-qt-share/CMakeLists.txt
+++ peony-extensions-cmake/peony-qt-share/CMakeLists.txt
@@ -25,4 +25,4 @@ add_library(${PROJECT_NAME} SHARED ${SRC_FILES} ${QRC_FILES})
 
 target_link_libraries(${PROJECT_NAME} Qt5::Core Qt5::Gui Qt5::Widgets Qt5::Concurrent ${PEONY_LIBRARIES})
 
-install(TARGETS peony-qt-share DESTINATION /usr/lib/peony-qt-extensions)
+install(TARGETS peony-qt-share DESTINATION /usr/lib64/peony-qt-extensions)
diff --git peony-extensions-qmake/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro peony-extensions-qmake/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro
index da9f48d..74a8208 100644
--- peony-extensions-qmake/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro
+++ peony-extensions-qmake/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro
@@ -37,7 +37,7 @@ HEADERS += \
         gtk2-style-enhanced-plugin_global.h
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
 
diff --git peony-extensions-qmake/peony-qt-create-link/peony-qt-create-link.pro peony-extensions-qmake/peony-qt-create-link/peony-qt-create-link.pro
index 0509d03..6802c70 100644
--- peony-extensions-qmake/peony-qt-create-link/peony-qt-create-link.pro
+++ peony-extensions-qmake/peony-qt-create-link/peony-qt-create-link.pro
@@ -36,7 +36,7 @@ HEADERS += \
         peony-qt-create-link_global.h 
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
 
diff --git peony-extensions-qmake/peony-qt-engrampa-menu-plugin/peony-qt-engrampa-menu-plugin.pro peony-extensions-qmake/peony-qt-engrampa-menu-plugin/peony-qt-engrampa-menu-plugin.pro
index 25147c5..a9b0b94 100644
--- peony-extensions-qmake/peony-qt-engrampa-menu-plugin/peony-qt-engrampa-menu-plugin.pro
+++ peony-extensions-qmake/peony-qt-engrampa-menu-plugin/peony-qt-engrampa-menu-plugin.pro
@@ -38,7 +38,7 @@ HEADERS += \
         peony-qt-engrampa-menu-plugin_global.h 
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
 
diff --git peony-extensions-qmake/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro peony-extensions-qmake/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro
index d7e0a48..3a47c2b 100644
--- peony-extensions-qmake/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro
+++ peony-extensions-qmake/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro
@@ -37,7 +37,7 @@ HEADERS += \
 TRANSLATIONS += translations/peony-qt-mate-terminal-extension_zh_CN.ts
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
 
