diff --git libpeony-qt/libpeony-qt.pro libpeony-qt/libpeony-qt.pro
index 90fae55..50bca12 100644
--- libpeony-qt/libpeony-qt.pro
+++ libpeony-qt/libpeony-qt.pro
@@ -37,7 +37,7 @@ DESTDIR += $$PWD
 
 unix {
     target.path = $$[QT_INSTALL_LIBS]
-    #target.path = /usr/lib
+    #target.path = /usr/lib64
     INSTALLS += target
 
     header.path = /usr/include/peony-qt
diff --git libpeony-qt/plugin-manager.cpp libpeony-qt/plugin-manager.cpp
index 74d70c4..255f0c7 100644
--- libpeony-qt/plugin-manager.cpp
+++ libpeony-qt/plugin-manager.cpp
@@ -54,7 +54,7 @@ PluginManager::PluginManager(QObject *parent) : QObject(parent)
     DirectoryViewFactoryManager2::getInstance();
     PreviewPageFactoryManager::getInstance();
 
-    QDir pluginsDir("/usr/lib/peony-qt-extensions");
+    QDir pluginsDir("/usr/lib64/peony-qt-extensions");
     pluginsDir.setFilter(QDir::Files);
 
     qDebug()<<pluginsDir.entryList().count();
diff --git peony-qt-desktop/desktop-menu-plugin-manager.cpp peony-qt-desktop/desktop-menu-plugin-manager.cpp
index 583f031..ef97b55 100644
--- peony-qt-desktop/desktop-menu-plugin-manager.cpp
+++ peony-qt-desktop/desktop-menu-plugin-manager.cpp
@@ -54,7 +54,7 @@ DesktopMenuPluginManager::~DesktopMenuPluginManager()
 void DesktopMenuPluginManager::loadAsync()
 {
     qDebug()<<"test start";
-    QDir pluginsDir("/usr/lib/peony-qt-extensions");
+    QDir pluginsDir("/usr/lib64/peony-qt-extensions");
     pluginsDir.setFilter(QDir::Files);
     Q_FOREACH(QString fileName, pluginsDir.entryList(QDir::Files)) {
         QPluginLoader pluginLoader(pluginsDir.absoluteFilePath(fileName));
diff --git peony-qt-plugin-test/directory-view-extensions-plugin-test/directory-view-extensions-plugin-test.pro peony-qt-plugin-test/directory-view-extensions-plugin-test/directory-view-extensions-plugin-test.pro
index 0efa1bd..c524035 100644
--- peony-qt-plugin-test/directory-view-extensions-plugin-test/directory-view-extensions-plugin-test.pro
+++ peony-qt-plugin-test/directory-view-extensions-plugin-test/directory-view-extensions-plugin-test.pro
@@ -36,6 +36,6 @@ HEADERS += \
         directory-view-extensions-plugin-test_global.h 
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
diff --git peony-qt-plugin-test/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro peony-qt-plugin-test/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro
index 7697a50..5ce096e 100644
--- peony-qt-plugin-test/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro
+++ peony-qt-plugin-test/gtk2-style-enhanced-plugin/gtk2-style-enhanced-plugin.pro
@@ -38,6 +38,6 @@ HEADERS += \
 DESTDIR += ../../testdir
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
diff --git peony-qt-plugin-test/peony-qt-menu-plugin-example/peony-qt-menu-plugin-example.pro peony-qt-plugin-test/peony-qt-menu-plugin-example/peony-qt-menu-plugin-example.pro
index 48a05c8..cae085a 100644
--- peony-qt-plugin-test/peony-qt-menu-plugin-example/peony-qt-menu-plugin-example.pro
+++ peony-qt-plugin-test/peony-qt-menu-plugin-example/peony-qt-menu-plugin-example.pro
@@ -34,6 +34,6 @@ HEADERS += \
 DESTDIR += ../../testdir
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
diff --git peony-qt-plugin-test/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro peony-qt-plugin-test/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro
index 4eb79ea..a765f81 100644
--- peony-qt-plugin-test/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro
+++ peony-qt-plugin-test/peony-qt-menu-plugin-mate-terminal/peony-qt-menu-plugin-mate-terminal.pro
@@ -40,7 +40,7 @@ DESTDIR = ../../testdir
 TRANSLATIONS += translations/peony-qt-mate-terminal-extension_zh_CN.ts
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
 
diff --git peony-qt-plugin-test/peony-qt-preview-file-content-plugin/peony-qt-preview-file-content-plugin.pro peony-qt-plugin-test/peony-qt-preview-file-content-plugin/peony-qt-preview-file-content-plugin.pro
index 0f8cb2c..336a01e 100644
--- peony-qt-plugin-test/peony-qt-preview-file-content-plugin/peony-qt-preview-file-content-plugin.pro
+++ peony-qt-plugin-test/peony-qt-preview-file-content-plugin/peony-qt-preview-file-content-plugin.pro
@@ -36,6 +36,6 @@ HEADERS += \
 DESTDIR += ../../testdir
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
diff --git peony-qt-plugin-test/properties-window-tab-page-plugin-example/properties-window-tab-page-plugin-example.pro peony-qt-plugin-test/properties-window-tab-page-plugin-example/properties-window-tab-page-plugin-example.pro
index 72b0ae4..5be951e 100644
--- peony-qt-plugin-test/properties-window-tab-page-plugin-example/properties-window-tab-page-plugin-example.pro
+++ peony-qt-plugin-test/properties-window-tab-page-plugin-example/properties-window-tab-page-plugin-example.pro
@@ -34,6 +34,6 @@ HEADERS += \
 DESTDIR = ../../testdir
 
 unix {
-    target.path = /usr/lib/peony-qt-extensions
+    target.path = /usr/lib64/peony-qt-extensions
     INSTALLS += target
 }
