diff --git pam-biometric/CMakeLists.txt pam-biometric/CMakeLists.txt
index 6f7ebdb..3ec1420 100644
--- pam-biometric/CMakeLists.txt
+++ pam-biometric/CMakeLists.txt
@@ -33,7 +33,7 @@ target_link_libraries(pam_biometric ${PAM_LIB})
 #去除lib前缀
 set_target_properties(pam_biometric PROPERTIES PREFIX "")
 
-install(TARGETS pam_biometric DESTINATION /lib/security)
+install(TARGETS pam_biometric DESTINATION /usr/lib64/security)
 install(FILES pam-configs/pam-biometric DESTINATION /usr/share/pam-configs)
 install(FILES data/ukui-biometric.conf DESTINATION /etc/biometric-auth)
 install(FILES
diff --git polkit-agent/CMakeLists.txt polkit-agent/CMakeLists.txt
index b489806..8388cc7 100644
--- polkit-agent/CMakeLists.txt
+++ polkit-agent/CMakeLists.txt
@@ -40,7 +40,7 @@ target_link_libraries(polkit-ukui-authentication-agent-1
     BioAuthWidgets
     )
 install(TARGETS polkit-ukui-authentication-agent-1
-    DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/ukui-polkit)
+    DESTINATION lib64/${CMAKE_LIBRARY_ARCHITECTURE}/ukui-polkit)
 
 # 翻译文件
 file(GLOB ts_files i18n_ts/*.ts)
