diff --git plugins/account/loginoptions/loginoptions.pro plugins/account/loginoptions/loginoptions.pro
index ca3b420..4f52ea1 100644
--- plugins/account/loginoptions/loginoptions.pro
+++ plugins/account/loginoptions/loginoptions.pro
@@ -17,7 +17,7 @@ include(../../pluginsComponent/pluginsComponent.pri)
 
 #DEFINES += QT_DEPRECATED_WARNINGS
 
-target.path = $$[QT_INSTALL_PREFIX]/lib/control-center/plugins/
+target.path = $$[QT_INSTALL_PREFIX]/lib64/control-center/plugins/
 INSTALLS += target
 
 SOURCES += \
diff --git plugins/account/userinfo/userinfo.pro plugins/account/userinfo/userinfo.pro
index 1b665ae..53b6105 100644
--- plugins/account/userinfo/userinfo.pro
+++ plugins/account/userinfo/userinfo.pro
@@ -22,7 +22,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          += -L/usr/lib/ -lcrypt
+LIBS          += -L/usr/lib64/ -lcrypt
 
 ##加载gio库和gio-unix库
 CONFIG        += link_pkgconfig \
diff --git plugins/devices/audio/audio.pro plugins/devices/audio/audio.pro
index 6eae727..4b09549 100644
--- plugins/devices/audio/audio.pro
+++ plugins/devices/audio/audio.pro
@@ -27,7 +27,7 @@ PKGCONFIG += libmatemixer \
 
 #DEFINES += QT_DEPRECATED_WARNINGS
 
-target.path = $$[QT_INSTALL_PREFIX]/lib/control-center/plugins/
+target.path = $$[QT_INSTALL_PREFIX]/lib64/control-center/plugins/
 INSTALLS += target
 
 SOURCES += \
diff --git plugins/devices/keyboard/keyboard.pro plugins/devices/keyboard/keyboard.pro
index f19b17f..3335d16 100644
--- plugins/devices/keyboard/keyboard.pro
+++ plugins/devices/keyboard/keyboard.pro
@@ -19,7 +19,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          +=  -L/usr/lib/ -lgsettings-qt
+LIBS          +=  -L/usr/lib64/ -lgsettings-qt
 
 CONFIG        += link_pkgconfig \
                  C++11
diff --git plugins/devices/mouse/mouse.pro plugins/devices/mouse/mouse.pro
index 26edae7..bf179e4 100644
--- plugins/devices/mouse/mouse.pro
+++ plugins/devices/mouse/mouse.pro
@@ -20,8 +20,8 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          +=  -L/usr/lib/  -lXi -lgsettings-qt
-#LIBS          +=  -L/usr/lib/   -lgsettings-qt
+LIBS          +=  -L/usr/lib64/  -lXi -lgsettings-qt
+#LIBS          +=  -L/usr/lib64/   -lgsettings-qt
 
 
 
diff --git plugins/devices/mousecontrol/mousecontrol.pro plugins/devices/mousecontrol/mousecontrol.pro
index c77de3f..15e57c3 100644
--- plugins/devices/mousecontrol/mousecontrol.pro
+++ plugins/devices/mousecontrol/mousecontrol.pro
@@ -14,13 +14,13 @@ CONFIG += plugin
 INCLUDEPATH += ../../.. \
                 /usr/include/kylinsdk \
 
-LIBS          +=  -L/usr/lib/ -lmouseclient  -ltouchpadclient -lXi -lgsettings-qt
+LIBS          +=  -L/usr/lib64/ -lmouseclient  -ltouchpadclient -lXi -lgsettings-qt
 
 DESTDIR = ../..
 
 include(../../pluginsComponent/pluginsComponent.pri)
 
-target.path = $$[QT_INSTALL_PREFIX]/lib/control-center/plugins/
+target.path = $$[QT_INSTALL_PREFIX]/lib64/control-center/plugins/
 INSTALLS += target
 
 
diff --git plugins/devices/shortcut/shortcut.pro plugins/devices/shortcut/shortcut.pro
index ed19d72..1073ee0 100644
--- plugins/devices/shortcut/shortcut.pro
+++ plugins/devices/shortcut/shortcut.pro
@@ -14,7 +14,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_ROOTDIR \
                  /usr/include/dconf
 
-LIBS          +=  -L/usr/lib/ -lgsettings-qt -ldconf
+LIBS          +=  -L/usr/lib64/ -lgsettings-qt -ldconf
 
 CONFIG +=  \
           link_pkgconfig \
diff --git plugins/devices/touchpad/touchpad.pro plugins/devices/touchpad/touchpad.pro
index 08f082a..c69be9d 100644
--- plugins/devices/touchpad/touchpad.pro
+++ plugins/devices/touchpad/touchpad.pro
@@ -21,8 +21,8 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-#LIBS          +=  -L/usr/lib/ -ltouchpadclient -lXi -lgsettings-qt
-LIBS          +=  -L/usr/lib/  -lXi -lgsettings-qt
+#LIBS          +=  -L/usr/lib64/ -ltouchpadclient -lXi -lgsettings-qt
+LIBS          +=  -L/usr/lib64/  -lXi -lgsettings-qt
 
 CONFIG        += link_pkgconfig \
                  C++11
diff --git plugins/messages-task/experienceplan/experienceplan.pro plugins/messages-task/experienceplan/experienceplan.pro
index 2e88542..30f89e6 100644
--- plugins/messages-task/experienceplan/experienceplan.pro
+++ plugins/messages-task/experienceplan/experienceplan.pro
@@ -15,7 +15,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          +=  -L/usr/lib/ -lgsettings-qt
+LIBS          +=  -L/usr/lib64/ -lgsettings-qt
 
 
 CONFIG += c++11
diff --git plugins/messages-task/multitask/multitask.pro plugins/messages-task/multitask/multitask.pro
index 3b20e36..8f7b8e1 100644
--- plugins/messages-task/multitask/multitask.pro
+++ plugins/messages-task/multitask/multitask.pro
@@ -17,7 +17,7 @@ include(../../pluginsComponent/pluginsComponent.pri)
 
 #DEFINES += QT_DEPRECATED_WARNINGS
 
-target.path = $$[QT_INSTALL_PREFIX]/lib/control-center/plugins/
+target.path = $$[QT_INSTALL_PREFIX]/lib64/control-center/plugins/
 INSTALLS += target
 
 SOURCES += \
diff --git plugins/messages-task/notice/notice.pro plugins/messages-task/notice/notice.pro
index d34a33d..6f6dc48 100644
--- plugins/messages-task/notice/notice.pro
+++ plugins/messages-task/notice/notice.pro
@@ -25,7 +25,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_ROOTDIR \
                  /usr/include/dconf
 
-LIBS          +=  -L/usr/lib/ -lgsettings-qt -ldconf
+LIBS          +=  -L/usr/lib64/ -lgsettings-qt -ldconf
 
 
 CONFIG += c++11
diff --git plugins/network/netconnect/netconnect.pro plugins/network/netconnect/netconnect.pro
index 7350fdb..c64434e 100644
--- plugins/network/netconnect/netconnect.pro
+++ plugins/network/netconnect/netconnect.pro
@@ -18,7 +18,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          +=  -L/usr/lib/ -lgsettings-qt
+LIBS          +=  -L/usr/lib64/ -lgsettings-qt
 #DEFINES += QT_DEPRECATED_WARNINGS
 
 SOURCES += \
diff --git plugins/network/proxy/proxy.pro plugins/network/proxy/proxy.pro
index 81e7892..3014d08 100644
--- plugins/network/proxy/proxy.pro
+++ plugins/network/proxy/proxy.pro
@@ -20,7 +20,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          += -L/usr/lib/ -lgsettings-qt
+LIBS          += -L/usr/lib64/ -lgsettings-qt
 
 ##加载gio库和gio-unix库，用于获取和设置enum类型的gsettings
 CONFIG        += link_pkgconfig \
diff --git plugins/personalized/desktop/desktop.pro plugins/personalized/desktop/desktop.pro
index 10f804c..ed604c5 100644
--- plugins/personalized/desktop/desktop.pro
+++ plugins/personalized/desktop/desktop.pro
@@ -30,7 +30,7 @@ INCLUDEPATH   +=  \
                  /usr/include/dconf \
 
 
-LIBS          += -L/usr/lib/ -lgsettings-qt -ldconf
+LIBS          += -L/usr/lib64/ -lgsettings-qt -ldconf
 
 #DEFINES += QT_DEPRECATED_WARNINGS
 
diff --git plugins/personalized/fonts/fonts.pro plugins/personalized/fonts/fonts.pro
index d091f9a..b4b941d 100644
--- plugins/personalized/fonts/fonts.pro
+++ plugins/personalized/fonts/fonts.pro
@@ -26,7 +26,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS += -L/usr/lib/ -lgsettings-qt
+LIBS += -L/usr/lib64/ -lgsettings-qt
 
 #DEFINES += QT_DEPRECATED_WARNINGS
 
diff --git plugins/personalized/screenlock/screenlock.pro plugins/personalized/screenlock/screenlock.pro
index 1a4d8e2..cdc887a 100644
--- plugins/personalized/screenlock/screenlock.pro
+++ plugins/personalized/screenlock/screenlock.pro
@@ -21,7 +21,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_ROOTDIR \
                  $$PROJECT_COMPONENTSOURCE \
 
-LIBS += -L/usr/lib/ -lgsettings-qt
+LIBS += -L/usr/lib64/ -lgsettings-qt
 
 
 ##加载gio库和gio-unix库
diff --git plugins/personalized/screensaver/screensaver.cpp plugins/personalized/screensaver/screensaver.cpp
index c45a509..cb73d86 100644
--- plugins/personalized/screensaver/screensaver.cpp
+++ plugins/personalized/screensaver/screensaver.cpp
@@ -443,7 +443,7 @@ void Screensaver::status_init(){
 //    delete publicdata;
 }
 
-#define UKUI_SCREENSAVER_BIN "/usr/lib/ukui-screensaver/ukui-screensaver-default"
+#define UKUI_SCREENSAVER_BIN "/usr/lib64/ukui-screensaver/ukui-screensaver-default"
 void Screensaver::startupScreensaver(){
     //关闭屏保
     closeScreensaver();
diff --git plugins/personalized/screensaver/screensaver.pro plugins/personalized/screensaver/screensaver.pro
index d23774b..2096b84 100644
--- plugins/personalized/screensaver/screensaver.pro
+++ plugins/personalized/screensaver/screensaver.pro
@@ -20,7 +20,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          += -L/usr/lib/ -lgsettings-qt
+LIBS          += -L/usr/lib64/ -lgsettings-qt
 
 ##加载gio库和gio-unix库
 CONFIG        += link_pkgconfig \
diff --git plugins/personalized/theme/theme.pro plugins/personalized/theme/theme.pro
index bbbdcdf..a03fb5c 100644
--- plugins/personalized/theme/theme.pro
+++ plugins/personalized/theme/theme.pro
@@ -21,7 +21,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          += -L/usr/lib/ -lgsettings-qt -lX11 -lXfixes -lXcursor
+LIBS          += -L/usr/lib64/ -lgsettings-qt -lX11 -lXfixes -lXcursor
 
 
 #DEFINES += QT_DEPRECATED_WARNINGS
diff --git plugins/personalized/wallpaper/wallpaper.pro plugins/personalized/wallpaper/wallpaper.pro
index 068328a..6e73822 100644
--- plugins/personalized/wallpaper/wallpaper.pro
+++ plugins/personalized/wallpaper/wallpaper.pro
@@ -20,7 +20,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_ROOTDIR \
                  $$PROJECT_COMPONENTSOURCE \
 
-LIBS += -L/usr/lib/ -lgsettings-qt
+LIBS += -L/usr/lib64/ -lgsettings-qt
 
 ##加载gio库和gio-unix库，用于获取和设置enum类型的gsettings
 CONFIG        += link_pkgconfig \
diff --git plugins/security-updates/recovery/recovery.pro plugins/security-updates/recovery/recovery.pro
index baf096f..d650095 100644
--- plugins/security-updates/recovery/recovery.pro
+++ plugins/security-updates/recovery/recovery.pro
@@ -17,7 +17,7 @@ include(../../pluginsComponent/pluginsComponent.pri)
 
 #DEFINES += QT_DEPRECATED_WARNINGS
 
-target.path = $$[QT_INSTALL_PREFIX]/lib/control-center/plugins/
+target.path = $$[QT_INSTALL_PREFIX]/lib64/control-center/plugins/
 INSTALLS += target
 
 
diff --git plugins/system/defaultapp/defaultapp.pro plugins/system/defaultapp/defaultapp.pro
index 7edcf94..8fe5f72 100644
--- plugins/system/defaultapp/defaultapp.pro
+++ plugins/system/defaultapp/defaultapp.pro
@@ -22,7 +22,7 @@ include(../../../env.pri)
 INCLUDEPATH   +=  \
                  $$PROJECT_ROOTDIR \
 
-#LIBS          += -L/usr/lib/ -ldefaultprograms \
+#LIBS          += -L/usr/lib64/ -ldefaultprograms \
 
 
 SOURCES += \
diff --git plugins/system/display/display.pro plugins/system/display/display.pro
index 3da6100..d247bbd 100644
--- plugins/system/display/display.pro
+++ plugins/system/display/display.pro
@@ -18,7 +18,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_COMPONENTSOURCE \
                  $$PROJECT_ROOTDIR \
 
-LIBS          += -L/usr/lib/ -lgsettings-qt
+LIBS          += -L/usr/lib64/ -lgsettings-qt
 
 
 SOURCES += \
diff --git plugins/system/power/power.pro plugins/system/power/power.pro
index 4d56652..d0139ed 100644
--- plugins/system/power/power.pro
+++ plugins/system/power/power.pro
@@ -11,7 +11,7 @@ INCLUDEPATH += \
                $$PROJECT_ROOTDIR \
 
 
-LIBS     += -L/usr/lib/ -lgsettings-qt
+LIBS     += -L/usr/lib64/ -lgsettings-qt
 
 
 FORMS += \
diff --git plugins/time-language/area/area.pro plugins/time-language/area/area.pro
index 6d797fe..e076731 100644
--- plugins/time-language/area/area.pro
+++ plugins/time-language/area/area.pro
@@ -13,7 +13,7 @@ INCLUDEPATH += ../../..
 TARGET = $$qtLibraryTarget(area)
 DESTDIR = ../../../pluginlibs
 
-LIBS += -L/usr/lib/ -lgsettings-qt
+LIBS += -L/usr/lib64/ -lgsettings-qt
 
 ##加载gio库和gio-unix库，用于处理时间
 CONFIG        += link_pkgconfig \
diff --git plugins/time-language/datetime/datetime.pro plugins/time-language/datetime/datetime.pro
index 319e8c1..759c8ef 100644
--- plugins/time-language/datetime/datetime.pro
+++ plugins/time-language/datetime/datetime.pro
@@ -22,7 +22,7 @@ INCLUDEPATH   +=  \
                  $$PROJECT_ROOTDIR \
 
 
-LIBS += -L/usr/lib/ -lgsettings-qt
+LIBS += -L/usr/lib64/ -lgsettings-qt
 
 ##加载gio库和gio-unix库，用于处理时间
 CONFIG        += link_pkgconfig \
diff --git shell/shell.pro shell/shell.pro
index ef051a0..4275a0c 100644
--- shell/shell.pro
+++ shell/shell.pro
@@ -25,7 +25,7 @@ DEFINES += QT_DEPRECATED_WARNINGS
 
 include(../env.pri)
 
-LIBS += -L/usr/lib/ -lX11 -lgsettings-qt
+LIBS += -L/usr/lib64/ -lX11 -lgsettings-qt
 
 #LIBS += -L$$[QT_INSTALL_LIBS] -lkylinssoclient
 
@@ -39,7 +39,7 @@ PKGCONFIG     += gio-2.0 \
 inst1.files += ukui-control-center.desktop
 inst1.path = /usr/share/applications
 inst2.files += $$PROJECT_ROOTDIR/pluginlibs/
-inst2.path = /usr/lib/control-center/
+inst2.path = /usr/lib64/control-center/
 inst3.files += $$PROJECT_ROOTDIR/data/org.ukui.control-center.keybinding.gschema.xml
 inst3.files += $$PROJECT_ROOTDIR/data/org.ukui.control-center.desktop.gschema.xml
 inst3.files += $$PROJECT_ROOTDIR/data/org.ukui.control-center.notice.gschema.xml
