diff --git src/pluginmanage.cpp src/pluginmanage.cpp
index c66b886..da35b84 100644
--- src/pluginmanage.cpp
+++ src/pluginmanage.cpp
@@ -29,7 +29,7 @@ static PluginManager *global_instance = nullptr;
 
 PluginManager::PluginManager(QObject *parent) : QObject(parent)
 {
-    QDir pluginsDir("/usr/lib/ukui-sidebar/sidebarSmallPlugins");
+    QDir pluginsDir("/usr/lib64/ukui-sidebar/sidebarSmallPlugins");
     pluginsDir.setFilter(QDir::Files);
 
     qDebug()<<pluginsDir.entryList().count();
diff --git src/plugins/clipboardPlugin/clipboardPlugin.pro src/plugins/clipboardPlugin/clipboardPlugin.pro
index 6cc6c38..f03f59b 100644
--- src/plugins/clipboardPlugin/clipboardPlugin.pro
+++ src/plugins/clipboardPlugin/clipboardPlugin.pro
@@ -68,6 +68,6 @@ TRANSLATIONS += bo.ts
 TRANSLATIONS += tr.ts
 
 unix {
-    target.path = /usr/lib/ukui-sidebar/sidebar_clipboardPlugin_plugins
+    target.path = /usr/lib64/ukui-sidebar/sidebar_clipboardPlugin_plugins
     INSTALLS += target
 }
diff --git src/plugins/notification_plugin/notification_plugin.pro src/plugins/notification_plugin/notification_plugin.pro
index 1a31862..af8b180 100644
--- src/plugins/notification_plugin/notification_plugin.pro
+++ src/plugins/notification_plugin/notification_plugin.pro
@@ -49,7 +49,7 @@ HEADERS += \
 DISTFILES += notification_plugin.json  
 
 unix {
-    target.path = /usr/lib/ukui-sidebar/notification
+    target.path = /usr/lib64/ukui-sidebar/notification
     INSTALLS += target
 
     translation.path = /usr/share/ukui-sidebar-notification
diff --git src/widget.cpp src/widget.cpp
index 55b476c..25b5815 100644
--- src/widget.cpp
+++ src/widget.cpp
@@ -114,7 +114,7 @@ Widget::~Widget()
 //加载通知中心插件
 bool Widget::loadNotificationPlugin()
 {
-    QDir pluginsDir("/usr/lib/ukui-sidebar/notification");
+    QDir pluginsDir("/usr/lib64/ukui-sidebar/notification");
     QPluginLoader pluginLoader(pluginsDir.absoluteFilePath("libnotification_plugin.so"));
 
     m_pNotificationPluginObject = pluginLoader.instance();
@@ -136,7 +136,7 @@ bool Widget::loadNotificationPlugin()
 //加载剪贴板插件
 int Widget::ListenClipboardSignal()
 {
-    QDir pluginsDir("/usr/lib/ukui-sidebar/sidebar_clipboardPlugin_plugins");
+    QDir pluginsDir("/usr/lib64/ukui-sidebar/sidebar_clipboardPlugin_plugins");
     pluginsDir.setFilter(QDir::Files);
 
     QPluginLoader pluginLoader(pluginsDir.absoluteFilePath("libclipboardPlugin.so"));
