diff --git data/Makefile.am data/Makefile.am
index 413e128..1e841bb 100644
--- data/Makefile.am
+++ data/Makefile.am
@@ -6,7 +6,7 @@ dist_dbusif_DATA = org.ukui.Biometric.xml
 dbusconfdir = $(sysconfdir)/dbus-1/system.d
 dist_dbusconf_DATA = org.ukui.Biometric.conf
 
-servicedir = /lib/systemd/system/
+servicedir = /usr/lib/systemd/system/
 dist_service_DATA = biometric-authentication.service
 
 initddir = /etc/init.d/
